layouts/plugins/editors/tinymce/field/tinymcebuilder.php

```diff
@@ -61,9 +61,11 @@
 // Add assets
 $wa->registerAndUseStyle('tinymce.skin', 'media/vendor/tinymce/skins/ui/oxide/skin.min.css')
 	->registerAndUseStyle('plg_editors_tinymce.builder', 'plg_editors_tinymce/tinymce-builder.css', [], [], ['tinymce.skin', 'dragula'])
-	->registerAndUseScript('plg_editors_tinymce.builder', 'plg_editors_tinymce/tinymce-builder.js', [], ['type' => 'module'], ['core', 'dragula']);
+	->registerAndUseScript('plg_editors_tinymce.builder', 'plg_editors_tinymce/tinymce-builder.js', [], ['type' => 'module'], ['core', 'dragula'])
+	->useStyle('webcomponent.joomla-tab')
+	->useScript('webcomponent.joomla-tab');
 
-// Add TinyMCE language file to translate the buttons
+	// Add TinyMCE language file to translate the buttons
```

Comment:
```suggestion
// Add TinyMCE language file to translate the buttons
```
Align comment with `if` (or alternatively move the comment inside the `if`.

Reply:
that must have been by accident